modules/ding_webtrekk/ding_webtrekk.js

```diff
@@ -21,13 +20,13 @@
   var DING_WEBTREKK_PARAMETER_CAROUSEL_NEXT = 59;
   var DING_WEBTREKK_PARAMETER_CAROUSEL_PREV = 60;
 
-  var appendQueryParameter = function(url, key, value) {
```

Comment:
Why do we need to name our functions when we also assign them to variables?

Reply:
So when it don't work the console will write which function the error happened in a not just anonymous function.

It makes debugging easier when error message gets copied into a ticket.

Reply:
Thanks for elaborating. I think that is a totally valid argument. In that case: Have you considered whether you need to assign `function appendQueryParameter` to a variable at all?

I think it should be possible to declare the named functions without the assignment and then call them as you would normally.

Reply:
The issue here is that they are "exported" into global "Drupal" space in line 43 (https://github.com/ding2/ding2/blob/74e40bb8b9c0d36c897d1359598dacb4d20e243d/modules/ding_webtrekk/ding_webtrekk.js#L43) if they where not I would have removed the assignment.

Reply:
It's perfectly fine and normal to have a named function expression: https://kangax.github.io/nfe/#named-expr

Reply:
@holt83 I am not arguing against it.

As I see it @cableman is trying to [improve debugging for named functions](https://kangax.github.io/nfe/#names-in-debuggers) while my suggestion was then to simplify the existing code.

I see that the simplification is not possible so lets just roll with this.

Reply:
@kasperg I think that is the way to go!

Maybe we could also prefix the function names with dingWebtrekk to make it more clear that this is ding_webtrekk functions when they appear in debugging. But it's a minor one.